packages/core/src/components/menu-category/menu-category.tsx

```diff
@@ -151,8 +151,19 @@ export class MenuCategory {
   }
 
   private onNestedItemsChanged(mutations?: MutationRecord[]) {
+    const oldNestedItemsLength: number = this.nestedItems.length;
```

Comment:
i think we dont need this
```suggestion
    const oldNestedItemsLength = this.nestedItems.length;
```